foreman.fc

```diff
@@ -39,8 +39,8 @@
 
 # Passenger SCL file contexts
 
-/opt/rh/ruby193/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
-/opt/rh/ruby193/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/opt/theforeman/tfm/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/agents/.* -- gen_context(system_u:object_r:passenger_exec_t,s0)
+/opt/theforeman/tfm/root/usr/(share/gems/gems|lib64/gems/exts)/passenger-.*/helper-scripts/prespawn -- gen_context(system_u:object_r:passenger_exec_t,s0)
```

Comment:
No, we've always used our own build of it and will continue to for now at least.  There is a passenger40 collection, but adopting that's a whole other set of changes!